nixos/modules/services/backup/syncoid.nix

```diff
@@ -89,6 +26,16 @@ in
 
     package = lib.mkPackageOptionMD pkgs "sanoid" {};
 
+    nftables.enable = mkEnableOption (lib.mdDoc ''
+      nftables integration.
```

Comment:
Thanks for the clarifying description.
I've not renamed `enable` to `maintainUidSet` though, because IIRC there's a pattern throughout NixOS where integration into other services is just an unspecific `enable`.